src/app/services/infokit.service.ts

```diff
@@ -0,0 +1,25 @@
+import { Injectable } from '@angular/core';
+import { Http, Headers, Response } from '@angular/http';
+import { Observable } from 'rxjs/Observable';
+import { environment } from '../../environments/environment';
+import { APIService } from './api.service';
+import 'rxjs/Rx';
+
+@Injectable()
+export class InfokitService {
+
+    private _getInfokitActive = environment.baseURL + environment.apiEndpoint + 'infokitactive';
```

Comment:
The file is like this, 
	baseURL: 'http://localhost:3000/',
	apiEndpoint: 'api/',
	authEndpoint: 'auth/',
	regEndPoint: 'registration/'

We change the second variable and when the environment changes from localhost we will just have to change one variable to make it work. This was my thought about it.